app/test/simulation_test.go

```diff
@@ -0,0 +1,108 @@
+package app_test
+
+import (
+	"os"
+	"testing"
+
+	"github.com/celestiaorg/celestia-app/app"
+	"github.com/celestiaorg/celestia-app/app/encoding"
+	"github.com/cosmos/cosmos-sdk/baseapp"
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/simapp"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	simulationtypes "github.com/cosmos/cosmos-sdk/types/simulation"
+	"github.com/cosmos/cosmos-sdk/x/simulation"
+	"github.com/stretchr/testify/require"
+	abci "github.com/tendermint/tendermint/abci/types"
+)
+
+func init() {
+	simapp.GetSimulatorFlags()
+}
+
+type SimApp interface {
+	GetBaseApp() *baseapp.BaseApp
+	AppCodec() codec.Codec
+	SimulationManager() *module.SimulationManager
+	ModuleAccountAddrs() map[string]bool
+	Name() string
+	LegacyAmino() *codec.LegacyAmino
+	BeginBlocker(ctx sdk.Context, req abci.RequestBeginBlock) abci.ResponseBeginBlock
+	EndBlocker(ctx sdk.Context, req abci.RequestEndBlock) abci.ResponseEndBlock
+	InitChainer(ctx sdk.Context, req abci.RequestInitChain) abci.ResponseInitChain
+}
+
+//var defaultConsensusParams = &abci.ConsensusParams{
+//	Block: &abci.BlockParams{
+//		MaxBytes: 200000,
+//		MaxGas:   2000000,
+//	},
+//	Evidence: &tmproto.EvidenceParams{
+//		MaxAgeNumBlocks: 302400,
+//		MaxAgeDuration:  504 * time.Hour, // 3 weeks is the max duration
+//		MaxBytes:        10000,
+//	},
+//	Validator: &tmproto.ValidatorParams{
+//		PubKeyTypes: []string{
+//			tmtypes.ABCIPubKeyTypeEd25519,
+//		},
+//	},
+//}
+
+// BenchmarkSimulation run the chain simulation
+// Running using starport command:
+// `starport chain simulate -v --numBlocks 200 --blockSize 50`
+// Running as go benchmark test:
+// `go test -benchmem -run=^$ -bench ^BenchmarkSimulation ./app -NumBlocks=200 -BlockSize 50 -Commit=true -Verbose=true -Enabled=true`
```

Comment:
[non blocking question]
Do you know if simapp is using ABCI++? if not, the results of this could be misleading and not as useful as we might hope.

How can we see these results? do we just run them locally? What is blockSize 50?